.drone.yml

```diff
@@ -53,12 +53,13 @@ pipeline:
   upload-serenity-report:
     image: plugins/s3
     secrets: [ s3_bucket_name, aws_access_key_id, aws_secret_access_key ]
-    region: [ eu-west-2 ]
-    bucket: cs-test-s3
-    access_key: ${AWS_ACCESS_KEY_ID}
-    secret_key: ${AWS_SECRET_ACCESS_KEY}
-    source: target/site/serenity/**/*
-    target: /reports/$${DRONE_BUILD_NUMBER}/
+    settings:
+      region: [ eu-west-2 ]
```

Comment:
Is this supposed to be an array?

Reply:
Probably not. Please create a suggestion and I will commit.

Reply:
Suggestion commited.